CHANGELOG.md

```diff
@@ -1,5 +1,26 @@
 ## 3.63.0 (Unreleased)
 
+FEATURES:
+* **New Data Source:** `google_monitoring_istio_canonical_service` ([#8789](https://github.com/hashicorp/terraform-provider-google/pull/8789))
+* **New Resource:** `google_apigee_instance_attachment` ([#8795](https://github.com/hashicorp/terraform-provider-google/pull/8795))
+* **New Resource:** `google_gke_hub_membership` ([#8755](https://github.com/hashicorp/terraform-provider-google/pull/8755))
+* **New Resource:** `google_tags_tag_value` ([#8800](https://github.com/hashicorp/terraform-provider-google/pull/8800))
+
+IMPROVEMENTS:
+* added support for Apple silicon chip (updated to go 1.16) ([#3057](https://github.com/hashicorp/terraform-provider-google-beta/pull/3057))
+* container: 
+  * added support for GKE Autopilot in `google_container_cluster`([#3101](https://github.com/hashicorp/terraform-provider-google-beta/pull/3101))
+  * added `enable_l4_ilb_subsetting` (beta) and `private_ipv6_google_access` fields to `google_container_cluster` ([#3095](https://github.com/hashicorp/terraform-provider-google-beta/pull/3095))
```

Comment:
These are google-beta links + the formatting is a bit off

Reply:
Oh I see, it's all a block for container

Reply:
yesssssir